tests/src/test/scala/cats/tests/OneAndTests.scala

```diff
@@ -47,6 +47,14 @@ class OneAndTests extends CatsSuite {
   checkAll("NonEmptyList[Int]", ComonadTests[NonEmptyList].comonad[Int, Int, Int])
   checkAll("Comonad[NonEmptyList[A]]", SerializableTests.serializable(Comonad[NonEmptyList]))
 
+  test("Show is not empty and is formatted as expected") {
+    forAll { (nel: NonEmptyList[Int]) =>
+      nel.show.nonEmpty should === (true)
+      nel.show.startsWith("OneAnd") should === (true)
```

Comment:
Unsure if we want to test specifically for "OneAnd" but I can see there being value in testing if the `.show` of the head is in the string or something